zapisy/apps/theses/models.py

```diff
@@ -125,6 +126,12 @@ def is_supporting_advisor_assigned(self, user):
     def is_among_advisors(self, user):
         return self.is_mine(user) or self.is_supporting_advisor_assigned(user)
 
+    def is_user_privileged_for_thesis(self, user):
+        return self.is_among_advisors(user) or user.is_staff or is_theses_board_member(user)
+
+    def is_user_allowed_to_generate_declarations(self, user):
```

Comment:
Oczywiście fajnie jest nazywać takie metody od `is_` ale może możemy odejść od tej zasady dla zwięzłości? (To **nie** jest pytanie retoryczne.)
```suggestion
    def can_user_generate_declarations(self, user):
```

Reply:
Jasna sprawa